api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -1164,6 +1164,10 @@ message UpstreamConnectionOptions {
 
   // If set then set SO_KEEPALIVE on the socket to enable TCP Keepalives.
   core.v3.TcpKeepalive tcp_keepalive = 1;
+
+  // If true, Envoy will make the interface name of the local address for the upstream connection
```

Comment:
Comments should not mention Envoy specifically, since it is not the only xDS client.